ParabolInc/parabol
# feat(jira-server): Always show provider row with contact us button

Fixes #6269 and adds some markup fixes

<img width="1442" alt="image" src="https://user-images.githubusercontent.com/466991/172358771-4eab81cb-247e-40ec-953c-419c0e1361b5.png">

<img width="1247" alt="image" src="https://user-images.githubusercontent.com/466991/172359134-ab9bf056-3530-464f-a9ea-0b3eaca86394.png">


## Testing scenarios

- Create new org and team
- Go to integrations page
- See Jira server row is visible with `contact us` button
- Click `contact us`, see a new tab opened with a contact form with pre-filled subject
- See new event tracked in segment `Clicked Jira Server Request Button`
